jenkins/Jenkinsfile

```diff
@@ -393,7 +393,7 @@ pipeline {
                                 String daisyImage = "gcr.io/compute-image-tools/daisy:latest"
                                 sh """
                                     cd jenkins/gce-custom-images/cis-harden-images
-                                    git checkout $IMAGE_HARDENING_LIB_VERSION 
+                                    git fetch && git checkout $IMAGE_HARDENING_LIB_VERSION 
```

Comment:
nit:

this can be done by updating the .gitmodules file to track a specific branch via
`git config -f .gitmodules submodule.[submoduleName].branch [branchname]`
and then adding a `--remote` flag to the `submodule update` call

details were here https://git-scm.com/book/en/v2/Git-Tools-Submodules

not necessarily advocating to change it here since this works too, but I think it makes it more explicit that the intent is to update the submodule, as opposed to a fetch/checkout

Reply:
Oh wow..thanks for the info. I'm just hesitant to add any more git magic to it at this point, and that testing it thoroughly may not be so straightforward.

Reply:
yeah, I totally agree, if its working lets keep it, but a TIL for me when we have to do other submodule shenanigans